spec/features/merchant/dashboard_spec.rb

```diff
@@ -58,7 +58,7 @@
     it 'I can link to an order show page' do
       visit '/merchant'
 
-      click_link @order_2.id
+      click_link "#{@order_2.id}"
```

Comment:
trying to fix capybara warnings when I run rspec but it didn't really work. Decided to just keep going and not worry about it since it doesn't effect anything.